.github/workflows/summary_check.yaml

```diff
@@ -0,0 +1,20 @@
+name: summary check
+# This workflow is triggered on check runs
+on: 
+  status
+
+jobs:
+  summary_check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: summary_check
+        uses: sofisl/checkRunAction@v10
```

Comment:
nit, but when  we  move this to  its final place I would probably use hyphens, `check-run-action`.